src/search-scenario/azext_search_scenario/requests.py

```diff
@@ -0,0 +1,34 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+import json
+
+from ._clierror import ScenarioSearchError
+from .constants import MatchRule, SearchScope, SEARCH_SERVICE_URL
+
+
+def get_search_result_from_api(search_keyword, scope=SearchScope.All, match_rule=MatchRule.All, top_num=5):
```

Comment:
Please simplify the name of the parameters:
search_keyword -> keyword
top_num -> top

Reply:
Done